packages/react-positioning/src/usePopper.ts

```diff
@@ -388,14 +388,14 @@ export function usePopper(
       popperInstanceRef.current?.destroy();
       popperInstanceRef.current = null;
     };
-  }, [options.enabled, options.target]);
+  }, [handlePopperUpdate, options.enabled, options.target]);
```

Comment:
At a glance these look like cases where the omission of certain deps *may* be intentional, but it's hard for me to say since I'm not familiar with popper. So @ling1726 should definitely take another look at this before you merge.

Reply:
Thanks, yes I was planning to ping @ling1726 once the build was working in case there were other errors (I guess I wasn't seeing them locally because the lint results were cached and it didn't get invalidated when I changed the config).

Anyways, as far as I can tell, most of the omissions were probably intentional since (some) of these never change between renders. But it's also harmless to add them as deps in that case. I'll add a comment on the next one below since that does have some deps that could change functionality.

Reply:
in this case you can keep `handlePopperUpdate` in the deps, since it is created with `useEventCallback` and will always be referentially stable